### PR TITLE
Add docker files for different linux flavors

### PR DESCRIPTION
**Pull request by normanmaurer**
Motivation:

We build and release for different linux flavors. To make this easier provide docker files that can be used and also be used by others for a ready-to-use build env.

Modifications:

Add docker files for centos6 and debian7

Result:

Easier to release and build netty-tcnative.

### CODE REVIEW

**Comment by normanmaurer** [comment]
These are also the docker files I use for the docker images that are used to release tcnative :)

File: docker/Dockerfile-netty-tcnative-centos6 [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ openssl-devel \
+ perl \
+ tar \
+ unzip \
+ wget
+
+RUN mkdir $SOURCE_DIR
+WORKDIR $SOURCE_DIR
+
+RUN wget -q https://cmake.org/files/v3.8/cmake-$CMAKE_VERSION-Linux-x86_64.tar.gz && tar zxf cmake-$CMAKE_VERSION-Linux-x86_64.tar.gz && mv cmake-$CMAKE_VERSION-Linux-x86_64 /opt/ && echo 'PATH=/opt/cmake-$CMAKE_VERSION-Linux-x86_64/bin:$PATH' >> ~/.bashrc
```
Comment (Scottmitch) [inline docker/Dockerfile-netty-tcnative-centos6]: `3.8` and `CMAKE_VERSION` have some dependencies ... should there be some state shared here?

Comment (normanmaurer) [reply docker/Dockerfile-netty-tcnative-centos6]: @Scottmitch good idea!

**Comment by normanmaurer** [comment]
@Scottmitch PTAL again

**Comment by normanmaurer** [comment]
Cherry-picked into master (1c0ff1909fe65f82de2ffd48bfc3e81ebee0bf59) after squashed.

**Pull request closed without merging**